experimental/examples/chat-mpp/settings.gradle.kts

```diff
@@ -22,4 +22,3 @@ pluginManagement {
 
 rootProject.name = "chat-mpp"
 include(":client")
```

Comment:
It seems we can remove this module from this file

Reply:
Thanks, done